src/QRCoders.jl

```diff
@@ -196,7 +198,9 @@ function exportqrcode( message::AbstractString
                              mask=mask,
                              compact=compact)
 
-    if !endswith(path, ".png")
+    
+    if !contains("hello", r"\.[a-z0-9]+"i)
```

Comment:
Do you mean this?

```suggestion
    if !contains(path, r"\.[a-z0-9]+"i)
```

But I usually check the result of 'splitext'

Reply:
Yes! Thanks for catching that.